aiosmtpd/controller.py

```diff
@@ -8,13 +8,16 @@
 
 @public
 class Controller:
-    def __init__(self, handler, loop=None, hostname='::0', port=8025):
+    def __init__(self, handler, loop=None, hostname='::0', port=8025,
+                 thread_timeout=5):
```

Comment:
See my comment about the environment variable.  Maybe we don't need this constructor parameter?  If we want to keep it though, maybe `ready_timeout` is a better name?